pages/settings/index.vue

```diff
@@ -0,0 +1,333 @@
+          </v-card-text>
+        </v-card>
+      </v-col>
+      <v-col cols="12" md="6" lg="5" class="pt-0 pb-4">
+        <!-- User settings -->
+        <v-list two-line class="mb-4">
+          <v-list-item-group>
+            <v-list-item
+              v-for="userItem in userItems"
+              :key="userItem.name"
+              disabled
+            >
+              <v-list-item-avatar>
+                <v-icon v-text="userItem.icon" />
+              </v-list-item-avatar>
+              <v-list-item-content>
+                <v-list-item-title v-text="userItem.name" />
+                <v-list-item-subtitle v-text="userItem.description" />
+              </v-list-item-content>
+              <v-list-item-action>
+                <v-icon>mdi-chevron-right</v-icon>
+              </v-list-item-action>
+            </v-list-item>
+          </v-list-item-group>
+        </v-list>
+        <!-- Administrator settings -->
+        <v-list
+          v-if="$auth.user.Policy.IsAdministrator"
+          two-line
+          class="mb-4"
+          disabled
+        >
+          <v-list-item-group>
+            <v-list-item v-for="adminItem in adminItems" :key="adminItem.name">
+              <v-list-item-avatar>
+                <v-icon v-text="adminItem.icon" />
+              </v-list-item-avatar>
+              <v-list-item-content>
+                <v-list-item-title v-text="adminItem.name" />
+                <v-list-item-subtitle v-text="adminItem.description" />
+              </v-list-item-content>
+              <v-list-item-action>
+                <v-icon>mdi-chevron-right</v-icon>
+              </v-list-item-action>
+            </v-list-item>
+          </v-list-item-group>
+        </v-list>
+        <!-- Mobile links -->
+        <v-list v-if="$vuetify.breakpoint.mobile">
+          <v-list-item-group>
+            <v-list-item
+              v-for="linkItem in linkItems"
+              :key="linkItem.name"
+              :href="linkItem.link"
+              target="_blank"
+            >
+              <v-list-item-avatar>
+                <v-icon v-text="linkItem.icon" />
+              </v-list-item-avatar>
+              <v-list-item-content>
+                <v-list-item-title v-text="linkItem.name" />
+              </v-list-item-content>
+              <v-list-item-action>
+                <v-icon>mdi-open-in-new</v-icon>
+              </v-list-item-action>
+            </v-list-item>
+          </v-list-item-group>
+        </v-list>
+      </v-col>
+    </v-row>
+  </v-container>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import { mapActions } from 'vuex';
+import { isEmpty } from 'lodash';
+import { SystemInfo } from '@jellyfin/client-axios';
+import { version } from '~/package.json';
+import htmlHelper from '~/mixins/htmlHelper';
+
+export default Vue.extend({
+  mixins: [htmlHelper],
+  data() {
+    return {
+      systemInfo: {} as SystemInfo,
+      vueVersion: version,
+      userItems: [
```

Comment:
I think we should move all this block (linkItems, adminItems, userItems) to a mixin or somewhere else, so this page is as clean as possible. Imo this can get messy once we start adding more stuff in settings.

Reply:
Mixins are for reusable properties, methods, etc. These won't be reused elsewhere, so they don't fit into a mixin, imo.

Reply:
Nor do they really fit into another component, since (again) they won't be reused.

Reply:
> Imo this can get messy once we start adding more stuff in settings.

Do note that pretty much nothing will be added here (except maybe one or two links, which don't change much). Settings will all have their own sub-pages, completely separate from this.